user/mise/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "mise"
+pkgver = "2024.12.1"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [
+    "cargo",
+    "pkgconf",
+]
+makedepends = [
+    "libgit2-devel",
+    "openssl-devel",
+    "rust-std",
+    "zstd-devel",
+]
+checkdepends = ["bash"]
+pkgdesc = "Dev tools, env vars, task runner"
+maintainer = "ttyyls <contact@behri.org>"
+license = "MIT"
+url = "https://mise.jdx.dev"
+source = f"https://github.com/jdx/mise/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "0763599841970d9a60630fbd97301d7a06d9ce5c61d9da657fda432f2204cfae"
+
```

Comment:
Seems to build fine and pass tests with 
```python
hardening = ["vis", "cfi"]
```

Reply:
those are ignored in the cargo build style